change term scm to git (#1327)
iterative/dvc.org: content/docs/command-reference/commit.md

Data and pipelines are up to date.
```

> Nothing special is required, we simply `commit` to both the Git and DVC. Since

Reviewer: This change is correct but can you also please improve the sentence? It's not greatly written "to both the Git and DVC" doesn't sound great.

Author: Okay. Will do that.

> Instead of reproducing the pipeline for changes that do not produce a different

Reviewer 2: I think it should be ` produce different results` or ` produce a different result`.

> this pipeline is up to date, `dvc repro` will not do anything.

Reviewer: Hmmm... I'm not sure what this new sentence means either. Please try to understand the context and close the idea here. It's something along the lines of being able to force-register the changes to a stage/pipeline without reproducing it. It's best if you actually try the example to fully get the idea 🙂

Reviewer: May also want to try basic dvc run and dvc repro. See the examples in their cmd refs. Thanks

Author: Okay.

Reviewer: Moving to https://github.com/iterative/dvc.org/pull/1327#pullrequestreview-421506486

Data and pipelines are up to date.
```

> Instead of reproducing the pipeline for changes that do not produce a different
> output of pipeline stages, use `commit` to both git and DVC. Since this pipeline
> is up to date, `dvc repro` will not do anything.

Reviewer: Let's just simplify/generalize terminology, so complex context that aren't closely related to this cmd ref or example are avoided:
```suggestion
Instead of reproducing the pipeline for changes that do not produce a different
results, just use `commit` on both Git and DVC.
```
That last sentence was still redundant vs the previous paragraph before the code block...

Reviewer: But this is kind of still missing the main point I mentioned in https://github.com/iterative/dvc.org/pull/1327#discussion_r429603929

Reviewer: > It's something along the lines of being able to force-register the changes to a stage/pipeline without reproducing it.

Reviewer: p.s. the keyword being "force"

Author: I thought we would use the force option when committing to both DVC and Git if force pushing the changes was a requirement but it was not used in the example given. I am sorry, I think I got confused.

Reviewer: Hmmm I forgot there's a `--force` option. I guess using the term "force" here could be confusing after all. It's not really related to force-pushing to Git though.

But my point is the main idea for this last paragraph should be or should include that we are "forcing" DVC to commit the stage file changes even when the stage wasn't reproduced after it's dependencies change (because we know there's no need). Did you actually run the example commands to fully understand the example? In fact it may even need the `--force` option, I'm not sure. Please double check 🙂

Author: Yes I ran the commands. By using the -f option the prompt doesn't ask the user a yes/no question regarding the decision of committing. That is the only difference. Personally, I don't think it is necessary to put the force option but if that provides more clarity then we can do it.

Reviewer 2: I think it looks good already, the motivation for using commit in this example is provided a bit above - "we don't want to run repro because changes are not substantial ...".

(also, improving example itself probably out of scope for the ticket)
